### PR TITLE
Created index page

### PR DESCRIPTION
**Pull request by desmonduz**
I created an index page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome Page</title>
+</head>
+<body>
+    <h1>Hello world!</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo beatae deserunt eligendi aliquid, debitis aspernatur ipsum ducimus, libero sunt, nisi doloremque repellat fugit voluptatum! Iure aliquam consequuntur quia. Alias, vel.</p>
```
Comment (Ibrat1997) [inline index.html]: Please change it to bla bla bla

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <h2>About me</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Architecto porro ipsam, provident consequuntur. Eaque quod ipsum ipsam voluptatum cumque culpa, quisquam architecto sint voluptate eos minima commodi ea reiciendis neque.</p>
```
Comment (Ibrat1997) [inline index.html]: please change it to tra ta ta

Comment (desmonduz) [reply index.html]: Tra tra ta is not a good idea. Let's make it cha-cha-cha?

File: index.html (hunk 1 above)
Comment (desmonduz) [reply index.html]: I agree. I will fix it.

File: index.html (hunk 2 above)
Comment (Ibrat1997) [reply index.html]: okay I agree on cha-cha-cha

**Pull request merged**